doc/manual.rst

```diff
@@ -2592,6 +2592,17 @@ been declared with the `discardable`:idx: pragma:
 
   p(3, 4) # now valid
 
+however discardable pragma not working on template because template do AST substitute in place, for example
+
+.. code-block:: nim
+  {.push discardable .}
+  template example(): string = "https://nim-lang.org"
+  {.pop.}
+
+  example()
+
+This template will resolve into "https://nim-lang.org" which is string literal type, and since {.discardable.} don't apply to literals, the compiler will error.
```

Comment:
```suggestion
This template will resolve into "https://nim-lang.org" which is a string literal and since {.discardable.} doesn't apply to literals, the compiler will error.
```

Reply:
Thank you for help fix typo and grammar. Apologize for the mistake, english is not my native language.